src/EA.Weee.Core/Organisations/Base/OrganisationViewModel.cs

```diff
@@ -36,15 +37,31 @@ public class OrganisationViewModel : IValidatableObject
         [DisplayName("Organisation type")]
         public ExternalOrganisationType? OrganisationType { get; set; }
 
+        public bool IsPreviousSchemeMember { get; set; }
+
+        [StringLength(CommonMaxFieldLengths.ProducerRegistrationNumber)]
+        [DisplayName("Producer registration number")]
+        public virtual string ProducerRegistrationNumber { get; set; }
+
         public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
         {
-            return ExternalAddressValidator.Validate(Address.CountryId, Address.Postcode, "Address.CountryId", "Address.Postcode");
+            var results = new List<ValidationResult>();
+
+            if (IsPreviousSchemeMember && string.IsNullOrWhiteSpace(ProducerRegistrationNumber))
```

Comment:
@will-hope98  Cover these scenarios off with unit tests, there is the existing test class OrganisationViewModelTests